pyecm.py

```diff
@@ -18,29 +18,20 @@
 import sys
 import random
 
-try:
-   import psyco
-   psyco.full()
-   PSYCO_EXISTS = True
-except ImportError:
-   PSYCO_EXISTS = False
-
+INV_C = 1.4
 try: # Try to use gmpy
    from gmpy2 import isqrt as sqrt
    from gmpy2 import iroot as root
    from gmpy2 import gcd, invert, mpz, next_prime
    import gmpy2
-   GMPY_EXISTS = True
 except ImportError:
    try:
       from gmpy import gcd, invert, mpz, next_prime, sqrt, root
-      GMPY_EXISTS = True
    except ImportError:
-      GMPY_EXISTS = False
+      INV_C = 13.0
 
-if not GMPY_EXISTS:
+if INV_C == 13.0:
```

Comment:
Comparing to floats is not reliable. Further, I prefer the code flow as-is today, though I am fine removing the psyco support. Please just keep the **GMPY_EXISTS** code the way it is and change only the psyco-related code.

Reply:
It was reliable to me because you are setting it, it's not an input. Anyway let's keep it like this, your code has too many complaints from pep8, too many things to fix to make it a better approach to python coding that I don't want to spend time on it anymore.